packs/data/pathfinder-bestiary.db/ochre-jelly.json

```
@@ -265,7 +265,7 @@
                     "value": 1
                 },
                 "description": {
-                    "value": "<p>[[/r {1d8+3}[bludgeoning]+{1d4}[acid]]]{1d8+3 bludgeoning damage plus 2d4 acid damage}, @Check[type:fortitude|dc:23|basic:true] save</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.Constrict]</p>"
+                    "value": "<p>[[/r {1d8+3}[bludgeoning]+{1d4}[acid]]]{1d8+3 bludgeoning damage plus 1d4 acid damage}, @Check[type:fortitude|dc:23|basic:true] save</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.Constrict]</p>"
```

```suggestion
                    "value": "<p>[[/r {1d8+3}[bludgeoning]+{2d4}[acid]]]{1d8+3 bludgeoning damage plus 2d4 acid damage}, @Check[type:fortitude|dc:23|basic:true] save</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.Constrict]</p>"
```
Other way around turns out.